### DISCUSSION TITLE
New feature : isolation forest

### DISCUSSION THREAD

**Discussion by lusis-ai** [google/yggdrasil-decision-forests#80] [Ideas]
Hi,

Having isolation forest can be a great addition to YDF. Is it planned ?

**Comment by rstz** [comment]
I agree :) It's on our roadmap!

**Comment by rstz** [comment]
Update: Isolation Forests are now part of YDF!

[Here's a basic tutorial](https://ydf.readthedocs.io/en/latest/tutorial/anomaly_detection/) for Isolation Forests in YDF. There may still be some rough edges (please let us know!) and we're working on more features, e.g. support for categorical features.
